File: AccomodationServiceLibrary/Core/Model/Appointment.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AccomodationServiceLibrary.Core.Model
{
    public class Appointment
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }

        public DateTime From { get; set; }
        public DateTime To { get; set; }

        public double Price { get; set; }

        public bool PerGuest { get; set; }

    }
}

File: accommodation_service/domain/model/appointment.go
package model

import (
	"go.mongodb.org/mongo-driver/bson/primitive"
	"time"
)

type Appointment struct {
	ID       primitive.ObjectID `bson:"_id,omitempty"`
	From     time.Time          `bson:"from"`
	To       time.Time          `bson:"to"`
	Status   AppointmentStatus  `bson:"status"`
	Price    float64            `bson:"price"`
	PerGuest bool               `bson:"per_guest"`
}
